ROSCO/src/Controllers.f90

```diff
@@ -57,8 +57,21 @@ SUBROUTINE PitchControl(avrSWAP, CntrPar, LocalVar, objInst, DebugVar, ErrVar)
         LocalVar%PC_TF = interp1d(CntrPar%PC_GS_angles, CntrPar%PC_GS_TF, LocalVar%PC_PitComTF, ErrVar) ! TF gains (derivative filter) !NJA - need to clarify
         
         ! Compute the collective pitch command associated with the proportional and integral gains:
-        LocalVar%PC_PitComT = PIController(LocalVar%PC_SpdErr, LocalVar%PC_KP, LocalVar%PC_KI, LocalVar%PC_MinPit, LocalVar%PC_MaxPit, LocalVar%DT, LocalVar%BlPitch(1), LocalVar%piP, LocalVar%restart, objInst%instPI)
-        DebugVar%PC_PICommand = LocalVar%PC_PitComT
+	IF (LocalVar%iStatus == 0) THEN
```

Comment:
This is a core part of ROSCO that we would ideally leave untouched.  If you think there's a cleaner way of implementing the code, please try that.  I'll ponder it, as well.

Reply:
Are you referring to altering the Pitch Control subroutine? I tried multiple different ways to implement the feedforward control but this way gave the best results, though I can see how it is an integral part of ROSCO.

One idea I had was to implement the feedforward controller separately and combine the command, similar to how the floating feedback operates, though this didn't seem to deliver the benefits. I can try it again.

Reply:
I have updated this to work a little bit differently, please could you review and let me know your thoughts?